playground/terraform/infrastructure/cloudbuild-manual-setup/README.md

```diff
@@ -0,0 +1,159 @@
+and provide values as per example below:
+
+```
+project                       = "sandbox-playground-001"    # Your Project ID
+region                        = "us-central1"               # Your GCP region name where resources will be provisioned
+cloudbuild_service_account_id = "terraform-cloudbuild"      # Name of SA to be used by Cloud Build to deploy Playground
+```
+
+2. Run commands:
+
+```console
+# Creates new authentication configuration to GCP Project
+# Using user account in Requirements
+gcloud init
+
+# Acquires new user account credentials to use for Application Default Credentials
+gcloud auth application-default login
+
+# Navigate to ../01.setup folder
+cd beam/playground/terraform/cloudbuild-manual-setup/01.setup
+
+# Run terraform scripts
+terraform init -backend-config="bucket="YOUR_BUCKET_FOR_TF_STATE"
+terraform plan -var-file="../terraform.tfvars"
+terraform apply -var-file="../terraform.tfvars"
+
+```
+
+## 2. Connect GitHub repository and Cloud Build
+Follow [Connect to a GitHub repository](https://cloud.google.com/build/docs/automating-builds/github/connect-repo-github) to connect GitHub repository and Cloud Build.
+
+## 3. Setup the Google Cloud Build triggers
+
+The `playground/infrastructure/cloudbuild-manual-setup/02.builders` provisions:
+- Cloud Build trigger to build and deploy Beam Playground.
+
+#### Execute the module
+
+1. Append to `terraform.tfvars` file by path: `playground/terraform/cloudbuild-manual-setup/01.setup/terraform.tfvars`
+next variables:
+
+```
+github_repository_name        = "beam"                      # The name of the GitHub repo. Example: In https://github.com/example/foo is 'foo'
+github_repository_owner       = "ruslan-ikhsan"             # Owner of the GitHub repo. Example: In https://github.com/example/foo is 'example'.
+github_repository_branch      = "cloudbuild+playground"     # The name of GitHub repo branch
+```
+
+Final version of `terraform.tfvars` will look like:
+
+```
+project                       = "sandbox-playground-001"    # Your Project ID
+region                        = "us-central1"               # Your GCP region name where resources will be provisioned
+cloudbuild_service_account_id = "terraform-cloudbuild"      # Name of SA to be used by Cloud Build to deploy Playground
+github_repository_name        = "beam"                      # The name of the GitHub repo. Example: In https://github.com/example/foo is 'foo'
+github_repository_owner       = "ruslan-ikhsan"             # Owner of the GitHub repo. Example: In https://github.com/example/foo is 'example'.
+github_repository_branch      = "cloudbuild+playground"     # The name of GitHub repo branch
+```
+
+2. Run:
+
+```
+# Navigate to ../02.builders folder
+cd beam/playground/terraform/cloudbuild-manual-setup/02.builders
+# Run terraform scripts
+terraform init -backend-config="bucket="YOUR_BUCKET_FOR_TF_STATE"
+terraform plan -var-file="terraform.tfvars"
```

Comment:
Why is it -var-file="terraform.tfvars" here and -var-file="../terraform.tfvars" for 01.setup?
Double check and unify paths